sound/soc/sof/amd/acp.c

```diff
@@ -322,10 +369,21 @@ static int acp_init(struct snd_sof_dev *sdev)
 	return acp_reset(sdev);
 }
 
+static const struct sof_amd_acp_desc *get_chip_info(struct snd_sof_pdata *pdata)
+{
+	const struct sof_dev_desc *desc = pdata->desc;
+	const struct sof_amd_acp_desc *chip_info;
+
+	chip_info = desc->chip_info;
+
+	return chip_info;
```

Comment:
just return desc->chip_info should do right?

Reply:
that could also be inline if it is that small

Reply:
Or even a macro like `comp_get_drvdata` so it is generic

Reply:
we use an inline function in sound/soc/sof/intel/shim.h